[2022-12-04 15:59:07,409] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.world scheduled__2022-01-31T00:00:00+00:00 [queued]>
[2022-12-04 15:59:07,565] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.world scheduled__2022-01-31T00:00:00+00:00 [queued]>
[2022-12-04 15:59:07,576] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 15:59:07,578] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-04 15:59:07,581] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 15:59:07,850] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): world> on 2022-01-31 00:00:00+00:00
[2022-12-04 15:59:07,879] {standard_task_runner.py:52} INFO - Started process 399 to run task
[2022-12-04 15:59:08,045] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'world', 'scheduled__2022-01-31T00:00:00+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/first-dag.py', '--cfg-path', '/tmp/tmpdxq0juxk', '--error-file', '/tmp/tmpo83w85xm']
[2022-12-04 15:59:08,110] {standard_task_runner.py:80} INFO - Job 43: Subtask world
[2022-12-04 15:59:12,102] {task_command.py:369} INFO - Running <TaskInstance: first_dag.world scheduled__2022-01-31T00:00:00+00:00 [running]> on host fb90b48d1471
[2022-12-04 15:59:14,169] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=world
AIRFLOW_CTX_EXECUTION_DATE=2022-01-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-31T00:00:00+00:00
[2022-12-04 15:59:14,189] {logging_mixin.py:115} INFO - Airflow
[2022-12-04 15:59:14,210] {python.py:173} INFO - Done. Returned value was: None
[2022-12-04 16:02:17,502] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=world, execution_date=20220131T000000, start_date=20221204T155907, end_date=20221204T160217
[2022-12-04 16:02:19,079] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-04 16:02:21,510] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-12-04 16:42:22,400] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.world scheduled__2022-01-31T00:00:00+00:00 [queued]>
[2022-12-04 16:42:22,455] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.world scheduled__2022-01-31T00:00:00+00:00 [queued]>
[2022-12-04 16:42:22,456] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 16:42:22,457] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-04 16:42:22,457] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 16:42:22,530] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): world> on 2022-01-31 00:00:00+00:00
[2022-12-04 16:42:22,542] {standard_task_runner.py:52} INFO - Started process 934 to run task
[2022-12-04 16:42:22,593] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'world', 'scheduled__2022-01-31T00:00:00+00:00', '--job-id', '169', '--raw', '--subdir', 'DAGS_FOLDER/first-dag.py', '--cfg-path', '/tmp/tmp_1_hzsff', '--error-file', '/tmp/tmpd1thx20z']
[2022-12-04 16:42:22,600] {standard_task_runner.py:80} INFO - Job 169: Subtask world
[2022-12-04 16:42:22,858] {task_command.py:369} INFO - Running <TaskInstance: first_dag.world scheduled__2022-01-31T00:00:00+00:00 [running]> on host fb90b48d1471
[2022-12-04 16:42:23,278] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=world
AIRFLOW_CTX_EXECUTION_DATE=2022-01-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-31T00:00:00+00:00
[2022-12-04 16:42:23,282] {logging_mixin.py:115} INFO - Airflow
[2022-12-04 16:42:23,285] {python.py:173} INFO - Done. Returned value was: None
[2022-12-04 16:42:23,361] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=world, execution_date=20220131T000000, start_date=20221204T164222, end_date=20221204T164223
[2022-12-04 16:42:23,470] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-04 16:42:23,550] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
